test/data/submissions.js

```diff
@@ -100,54 +89,87 @@ export const extendedSubmissions = dataStore({
+        defaultFields.push(fields.group('/meta'), fields.string('/meta/instanceName'));
+      defaultFields.push(attachmentsExpected === 0
+        ? fields.string('/s')
+        : fields.binary('/b'));
+
+      // The lastSubmission property of the form will likely not match the
+      // submission.
+      extendedForms.createPast(1, { submissions: 1, fields: defaultFields });
+    }
+    const formVersion = formVersionOption ?? extendedForms.first();
+
+    if (submitterOption == null && extendedUsers.size === 0)
+      throw new Error('user not found');
+    const submitter = submitterOption ?? extendedUsers.first();
+
     const createdAt = !inPast
       ? new Date().toISOString()
       : fakePastDate([
         lastCreatedAt,
-        formVersion.publishedAt != null
-          ? formVersion.publishedAt
-          : formVersion.createdAt,
+        formVersion.publishedAt ?? formVersion.createdAt,
         submitter.createdAt
       ]);
+    const odata = randomOData(instanceId, formVersion._fields, {
+      ...partialOData,
+      __id: instanceId,
+      __system: {
+        submissionDate: createdAt,
+        updatedAt: null,
+        submitterId: submitter.id.toString(),
+        submitterName: submitter.displayName,
+        attachmentsPresent,
+        attachmentsExpected,
+        status,
+        reviewState,
+        deviceId,
+        edits,
+        formVersion: formVersion.version
+      }
+    });
     return {
       instanceId,
       deviceId,
-      formVersion: formVersion.version,
+      reviewState,
       submitterId: submitter.id,
       submitter: toActor(submitter),
       createdAt,
       updatedAt: null,
+      currentVersion: {
+        instanceId,
+        instanceName: hasInstanceName ? odata.meta.instanceName : null,
+        current: true,
+        formVersion: formVersion.version,
+        deviceId,
+        submitterId: submitter.id,
+        submitter: toActor(submitter),
+        createdAt
+      },
       // An actual submission JSON response does not have this property. We
       // include it here so that it is easy to match submission data and
       // metadata during testing.
-      _odata: odata(instanceId, formVersion._fields, {
-        ...partialOData,
-        __id: instanceId,
-        __system: {
-          submissionDate: createdAt,
-          updatedAt: null,
-          submitterId: submitter.id.toString(),
-          submitterName: submitter.displayName,
-          attachmentsPresent,
-          attachmentsExpected,
-          status,
-          reviewState,
-          deviceId,
-          edits,
-          formVersion: formVersion.version
-        }
-      })
+      _odata: odata
     };
   },
   sort: comparator((submission1, submission2) =>
     isBefore(submission2.createdAt, submission1.createdAt))
 });
 
-export const standardSubmissions = view(
-  extendedSubmissions,
-  omit(['submitter'])
-);
+export const standardSubmissions = view(extendedSubmissions, (extended) => {
```

Comment:
Delete

Reply:
It turns out that we do use `testData.standardSubmissions`, in tests of `SubmissionUpdateReviewState`.